src/cascadia/LocalTests_TerminalApp/TabTests.cpp

```diff
@@ -277,219 +277,227 @@ namespace TerminalAppLocalTests
-                    "historySize": 2
-                }
-            ]
-        })" };
-
-        VerifyParseSucceeded(settingsJson0);
-        auto settings0 = std::make_shared<CascadiaSettings>(false);
-        VERIFY_IS_NOT_NULL(settings0);
-        settings0->_ParseJsonString(settingsJson0, false);
-        settings0->LayerJson(settings0->_userSettings);
-        settings0->_ValidateSettings();
-
-        VerifyParseSucceeded(settingsJson1);
-        auto settings1 = std::make_shared<CascadiaSettings>(false);
-        VERIFY_IS_NOT_NULL(settings1);
-        settings1->_ParseJsonString(settingsJson1, false);
-        settings1->LayerJson(settings1->_userSettings);
-        settings1->_ValidateSettings();
-
-        const auto guid1 = Microsoft::Console::Utils::GuidFromString(L"{6239a42c-1111-49a3-80bd-e8fdd045185c}");
-        const auto guid2 = Microsoft::Console::Utils::GuidFromString(L"{6239a42c-2222-49a3-80bd-e8fdd045185c}");
-        const auto guid3 = Microsoft::Console::Utils::GuidFromString(L"{6239a42c-3333-49a3-80bd-e8fdd045185c}");
-
-        // This is super wacky, but we can't just initialize the
-        // com_ptr<impl::TerminalPage> in the lambda and assign it back out of
-        // the lambda. We'll crash trying to get a weak_ref to the TerminalPage
-        // during TerminalPage::Create() below.
-        //
-        // Instead, create the winrt object, then get a com_ptr to the
-        // implementation _from_ the winrt object. This seems to work, even if
-        // it's weird.
-        winrt::com_ptr<winrt::TerminalApp::implementation::TerminalPage> page{ nullptr };
-        _initializeTerminalPage(page, settings0);
-
-        auto result = RunOnUIThread([&page]() {
-            VERIFY_ARE_EQUAL(1u, page->_tabs.Size());
-        });
-        VERIFY_SUCCEEDED(result);
-
-        Log::Comment(L"Duplicate the first tab");
-        result = RunOnUIThread([&page]() {
-            page->_DuplicateTabViewItem();
-            VERIFY_ARE_EQUAL(2u, page->_tabs.Size());
-        });
-        VERIFY_SUCCEEDED(result);
-
-        Log::Comment(NoThrowString().Format(
-            L"Change the settings of the TerminalPage so the first profile is "
-            L"no longer in the list of profiles"));
-        result = RunOnUIThread([&page, settings1]() {
-            page->_settings = settings1;
-        });
-        VERIFY_SUCCEEDED(result);
-
-        Log::Comment(L"Duplicate the tab, and don't crash");
-        result = RunOnUIThread([&page]() {
-            page->_DuplicateTabViewItem();
-            VERIFY_ARE_EQUAL(2u, page->_tabs.Size(), L"We should gracefully do nothing here - the profile no longer exists.");
-        });
-        VERIFY_SUCCEEDED(result);
+        Log::Comment(L"This test regressed recently - it is temporarily disabled while GH#5169 is investigated");
+        Log::Result(WEX::Logging::TestResults::Skipped);
+        return;
+
+        // // * Create a tab with a profile with GUID 1

```

Comment:
Broadly I’d prefer an `#if 0` block because it doesn’t delta every single line, but.. idc enough to block for it